### DISCUSSION TITLE
2.8 condition in variable

### DISCUSSION THREAD

**Discussion by Herosteve0** [SkriptLang/Skript#6466] [General]
I recently updated skript from 2.6.4 to 2.8.2 and I can no longer set variables to conditions, for example

command /asd:
	trigger:
		set {_x} to 3
		set {_y} to 7
		set {_c} to {_x} > {_y}
		broadcast "%{_c}%"

wtih this being the error

![image](https://github.com/SkriptLang/Skript/assets/46967140/5e2912fc-744d-4066-aa55-82f25345ad83)

**Comment by bluelhf** [comment]
This was never the case, perhaps one of your add-ons broke as a result of the update?

**Reply by Herosteve0** [reply]
Probably. I updated SkBee along with Skript so maybe it has to do with that. Thanks for the suggestion!

**Comment by Herosteve0** [comment]
Okay, turns out it had nothing to do with Skript, it was SkQuery that required an update
